user-service/src/main/java/by/battle/userservice/exception/UserNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package by.battle.userservice.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+public class UserNotFoundException extends RuntimeException {
+
+    public UserNotFoundException(String userId) {
+        super("Can't find user with id : %s" + userId);
```

Comment:
%s will not work in it case,
"Can't find user with id : %s" + userId
will simply concatenate strings.
so in result you will have  string like this Can't find user with id : %sada1-0022-1111-222-222

you may use String.format and move this "Can't find user with id : %s" to static final variable